internal/cache/controller_obj_cache.go

```diff
@@ -3114,6 +3114,7 @@ func checkAndSetVRFFromNetwork(client *clients.AviClient, returnErr *error) bool
 	}
 	if lib.IsNodePortMode() {
 		utils.AviLog.Infof("Using global VRF for NodePort mode")
+		lib.SetVrf(utils.GlobalVRF)
```

Comment:
@aaha97, just a basic question. What will happen if the customer is in any public cloud with a non-admin tenant and wants to use global VRF?

Reply:
cuurently for public cloud, the getVrf function will be used. non admin tenant won't be able to use global vrf.